components/changelog/ChangelogTrigger.js

```diff
@@ -88,8 +89,10 @@ const ChangelogTrigger = props => {
             backgroundSize={backgroundSize}
             url="/static/images/flame-red.svg"
           />
-          <ChangelogNotificationDropdown />
-        </Dropdown>
+          <DismissibleMessage messageId="ChangelogTrigger.dropdownNotification">
```

Comment:
Would be nice to put this message in https://github.com/opencollective/opencollective-frontend/blob/b657bf1fd79e8ce3a260163b6001dc9071c169c8/lib/constants/dismissable-help-message.js#L4

Reply:
Ops, I missed that there was a constants file. Just added that there. 😄